src/sentry/static/sentry/app/locale.jsx

```diff
@@ -12,14 +14,21 @@ if (sessionStorage && sessionStorage.getItem('localeDebug') == '1') {
   LOCALE_DEBUG = true;
 }
 
+const markerCss = css`
+  background: #ff801790 !important;
```

Comment:
Do we need `!important` since you removed the css?

Reply:
hm, probably not actually